singlevenue/src/main/java/com/mappedin/examples/singlevenue/MainActivity.java

```diff
@@ -358,7 +435,15 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
                 }
             });
 
-            final Location[] locations = activeVenue.getLocations();
+            Location[] unsortedLocations = activeVenue.getLocations();
+            ArrayList<Location> locArrList = new ArrayList<>(Arrays.asList(unsortedLocations));
+            Collections.sort(locArrList, new Comparator<Location>() {
```

Comment:
looks like you already wrote a location comparor above, should reuse that rather than defining a new one